scrap.py

```diff
@@ -0,0 +1,59 @@
+import datetime
+import requests
+import time
+from facebook_scraper import get_posts, exceptions
+
+"""
+    here will be the main objects and methods to 
+    scrap the facebook pages the idea is init require 
+    two arguments the page name and how many page you 
+    want to scrape
+    
+"""
+
+DELAY_SECONDS = 2
+TEST_DB = {
+    'text': 'Everything you need to know to preorder Hogwarts Legacy, the upcoming open-world RPG set in '
+            'the wizarding world of Harry Potter, hundreds of years before the hero arrived.',
+    'photo': 'https://assets-prd.ignimgs.com/2022/03/18/hogwarts-legacy-4-1647622573681.jpeg?width=1280',
+    'time': datetime.datetime(2022, 3, 24, 16, 31, 9)}
```

Comment:
test constants files should never be in the real code you need to load them in a separate test file